analytic_base_department/tests/test_line_department.py

```diff
@@ -1,5 +1,5 @@
-# Copyright 2016 Yannick Vaucher (Camptocamp)
-# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+# -*- coding: utf-8 -*-
```

Comment:
Why are you removing copyright statements?
The utf-8 marker on Python files is redundant on Python3